src/components/common/GreenhouseEffectSettings.js

```diff
@@ -96,6 +126,9 @@ const GreenhouseEffectSettings = () => {
         onChange={(e, v) => onChange(v, 'carbonDioxide')}
         indent
         disabled={disabled}
+        valueLabelDisplay={isMarsOrVenus ? 'on' : 'auto'}
+        widerLabel={isMarsOrVenus}
```

Comment:
I haven't seen the component itself, but maybe it would be interesting either to call the prop `small` or `big` or go for some size directly (eg. `isMarsOrVenus ? 'big' : 'small'` )

Reply:
Interesting. I changed the prop name from `widerLabel` to `bigLabel`, which is clearer. But I think it's best to leave it as a boolean: If not `bigLabel`, then it is implied that it's the default label.